scripts/experimentation/ued.js

```diff
@@ -0,0 +1,186 @@
+/*
+ * Copyright 2022 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+function murmurhash3_32_gc(key, seed) {
+  var remainder = key.length & 3;
```

Comment:
Yes, there is :) 
This is an AEP script that may be used in other systems/solutions, and was provided like this to ensure browser compatibility.
This script is owned by AEP.

Reply:
as we're already using modern syntax everywhere else, i see no reason to exclude the AEP script. the syntax is supported by all major browsers since a long time: https://caniuse.com/let,const

Reply:
This script is the "Unified Experience Decisioning" engine owned by AEP team.
We are just adding it to the project. My understanding is that we shouldn't modify it. 
The file has been added like this to other projects.